### PR TITLE
add new vt decompose tool

### PR DESCRIPTION
**Pull request by bgruening**
@TorHou as promised ... would be great if you can review and test it.

### CODE REVIEW

**Comment by TorHou** [comment]
holy smokes @bgruening ... i don't even ... I was still collecting all the problems I will face getting the big RelinSys database, and you already start fixing them ..

**Comment by bgruening** [comment]
:) let me know what else is needed!

**Comment by bgruening** [comment]
@TorHou can we merge and push this?

File: tools/vt/vt_decompose.xml [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+        ln -s "${ infile }" infile.vcf &&
+
+
+        vt @BINARY@
+            #if str($output_format) == 'bcf':
+                -o decompose.bcf
+            #else:
+                -o decompose.vcf
+            #end if
```
Comment (TorHou) [inline tools/vt/vt_decompose.xml]: you don't necessarily need these. You could just call it decompose.tmp

File: tools/vt/vt_decompose.xml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        ## For some reason, the file move will randomly produce empty files.
+        ## Wait two seconds to let the system close file handlers and clean up.
+        sleep 2
+        &&
+
+        #if str($output_format) == 'bcf':
+            mv decompose.bcf "${ outfile }";
+        #else:
+            mv decompose.vcf "${ outfile }";
+        #end if
```
Comment (TorHou) [inline tools/vt/vt_decompose.xml]: mv decompose.tmp "${ outfile }";

**Comment by TorHou** [comment]
:+1: merge

**Comment by TorHou** [comment]
.shed.yml fehlt noch

**Pull request merged**